pymongo/mongo_client.py

```diff
@@ -645,6 +645,10 @@ def __init__(
         dbase = None
         opts = common._CaseInsensitiveDictionary()
         fqdn = None
+        # Raise an error when we have more than one URI
```

Comment:
This comment is redundant because the error message says the same thing.

Reply:
Done.